### ISSUE TITLE
[BUG] Multi line strings break vertical alignment

### ISSUE THREAD

**Issue by Delta456** [Delta456/box-cli-maker#13]
**Describe the bug**
The vertical alignment of the Box in the right will be broken if multi line strings are used.

**To Reproduce**
```go
Box := box.New(box.Config{Px: 2, Py: 5, Type: Single})
Box.Println(`Box CLI Maker 
		foo foobar`, `Highly Customized Terminal Box Maker  
		bar`)
```

**Expected behavior**
The vertical alignment shouldn't break.

**Screenshots / Logs**
If applicable, add screenshots or logs to help explain your problem.

![image](https://user-images.githubusercontent.com/28479139/97784127-4464e380-1bc2-11eb-96f1-dc92562b87f6.png)

**Versions (please complete the following information, if relevant):**
- Software version: v1.3.2
- OS: Windows 10
- Golang version: 1.15.3

**Comment by Delta456** [comment]
Fixed in #11

**Comment by Delta456** [comment]
This still isn't resolved. The vertical alignment will break for more than 1 tab.

**Comment by Delta456** [comment]
Fixed by #14
